docker-compose.yml

```diff
@@ -2,10 +2,16 @@ version: '3'
 
 services:
     rif-relay:
-        container_name: rif-relay-server
         build:
             context: .
+        entrypoint: 'node dist/commands/Start.js --config server-config.json'
         ports:
             - '8090:8090'
         volumes:
             - './environment:/srv/app/environment'
+        networks:
+            - rif-relay
+
+networks:
+    rif-relay:
+        driver: bridge
```

Comment:
I'm not sure if we need this. Why did you add it?

Reply:
Actually not necessary, forgot to remove it during debugging.